tests/foreman/cli/test_import.py

```diff
@@ -960,6 +962,221 @@ def test_import_repo_recovery(self, test_data):
                 Repository.info({'id': record['sat6']}).return_code, 0
             )
 
+    @data(*gen_import_cv_data())
+    def test_import_cv_default(self, test_data):
+        """@test: Import and enable all Content Views from the default data set
+        (predefined source)
+
+        @feature: Import Enable Content View
+        @assert: 3 Content Views imported and enabled
```

Comment:
Nitpick: split up the `@feature` and `@assert` sections:

        @feature: Import Enable Content View

        @assert: 3 Content Views imported and enabled

As is, this docstring is considered to be one big paragraph, and it can be written as so with no change in semantics:

        @feature: Import Enable Content View @assert: 3 Content Views imported
        and enabled